src/git/getCommits.ts

```diff
@@ -18,7 +18,7 @@ export default async function (
   const delimiter = `<--- ${randomBytes(64).toString("hex")} --->`;
   const gitParams = [
     "log",
-    `--pretty="format:%B%n-hash-%n%H ${delimiter}"`,
+    `--format=%B%n-hash-%n%H%n${delimiter}`,
```

Comment:
` ` replace by `%n`, which means `\n` when used, but 'tis fine: We trim the whitespace, so ` `, `\n`, same stuff.